### PR TITLE
[PWA-633] My Account: Address Book: Main View

### PR DESCRIPTION
**Pull request by tjwiebell**
## Description

As a shopper, when logged in and viewing the address book page I should see any of my stored addresses with any associated options. 

In-Scope: 

Create main address components for the Address book page for both desktop and mobile

Requirements: 

For each address card, shopper can view: 

Name
Address
Default Status
Edit (CTA)
Delete (CTA)
## Related Issue

* [[PWA-633](https://jira.corp.magento.com/browse/PWA-633)] My Account: Address Book: Main View

### Verification Steps

1. Compare Address book UI with mocks when -
    1. User does not have any address.
    2. User has one address
    3. User has 2 Address
    4. User has Multiple address
2. Verify Default address should have  Default label and should not have delete icon.
3. Verify address with minimum and maximum characters.
4. Verify page works from both left drawer and Top right Account menu.
5. Verify Existing user address gets displayed.
6. Verify after logout from Address book page user is taken to Home page.
7. Verify Mobile scroll functionality when there are many address.
8. Verify in all supported browsers and devices.
9. Checkout Edit shipping address should show similar data as of My address page.

## Checklist

* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.

### CODE REVIEW

**Comment by schensley** [comment]
@tjwiebell This looks good. I assume that the edit and delete functionality/flows are coming later. UX Approved.

File: packages/venia-ui/lib/components/AddressBookPage/addressCard.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+            <div className={classes.contentContainer}>
+                {defaultBadge}
+                <span className={classes.name}>{nameString}</span>
+                {streetRows}
+                <span>{additionalAddressString}</span>
+            </div>
+            <div className={classes.actionContainer}>
+                <LinkButton
+                    classes={{ root: classes.linkButton }}
+                    onClick={() => console.log('To be completed by PWA-634')}
```
Comment (supernova-at) [inline packages/venia-ui/lib/components/AddressBookPage/addressCard.js:54]: I know this is temporary but weirdly I'm not seeing these `console.log`s at all

**Review by supernova-at — APPROVED** [review]
Looks great!  Love the breakpoints.

Please add verification steps though 👍

File: packages/venia-ui/lib/components/AddressBookPage/addressCard.js (hunk 1 above)
Comment (tjwiebell) [reply packages/venia-ui/lib/components/AddressBookPage/addressCard.js:54]: I see them on a local in watch mode, but would expect terser to remove these from prod/PR builds. Was this on the PR deployed instance?

**Review by supernova-at — APPROVED** [review]

File: packages/venia-ui/lib/components/AddressBookPage/addressCard.js (hunk 1 above)
Comment (supernova-at) [reply packages/venia-ui/lib/components/AddressBookPage/addressCard.js:69]: Yep, it was. Good call

**Comment by dpatil-magento** [comment]
@tjwiebell Some observations - 
1. Default address is not always first in the list. (Try editing from checkout page).
image

2. There is slight delay in updating all addresses.
<a href="https://gyazo.com/ce618d1e67e4fe8098075a4e7b4f1ffa">Image from Gyazo</a>

3. Phone number is not displayed.

4. Country code (US, GB) is displayed instead of name.

**Comment by tjwiebell** [comment]
@dpatil-magento 

Reviewed these items with UX and all should be resolved in a626299. Updated mocks have been attached in JIRA. I'm not sure what order we want to do re-approvals in, but this will need another full round (UX, Code, QA).

**Comment by dpatil-magento** [comment]
QA Approved.  Phone number is not formatted per mocks but I will leave that call to UX.

**Review by supernova-at — APPROVED** [review]

**Comment by schensley** [comment]
@dpatil-magento I've reviewed the PR instance with the display of full name of Country and the additional display of a collected phone number. UX approved. Thanks.

**Review by supernova-at — APPROVED** [review]

**Pull request merged**